doc/code/overview.rst

```diff
@@ -0,0 +1,91 @@
+| :ref:`pennylane.configuration |  Contains methods of how to save a PennyLane configuration,                          |
+| <configuration>`              |  for example the number of shots for a device.                                       |
++-------------------------------+--------------------------------------------------------------------------------------+
+| :ref:`pennylane.decorator     |  Defines the ``qnode()`` decorator function,                                         |
+| <decorator>`                  |  which turns quantum functions into QNodes.                                          |
++-------------------------------+--------------------------------------------------------------------------------------+
+| :ref:`pennylane.init          |  Contains parameter initialization functions.                                        |
+| <par_init>`                   |                                                                                      |
++-------------------------------+--------------------------------------------------------------------------------------+
+| :ref:`pennylane.measure       |  Introduces different results to extract from a measurement,                         |
+| <measure>`                    |  i.e. :func:`expval`, :func:`var` and :func:`samples`.                               |
++-------------------------------+--------------------------------------------------------------------------------------+
+| :ref:`pennylane.ops <ops>`    |  Contains specific operations for quantum circuits,                                  |
+|                               |  such as the :func:`Kerr` gate and the :func:`PauliZ` observable.                    |
++-------------------------------+--------------------------------------------------------------------------------------+
+| :ref:`pennylane.optimize      |  Contains the optimizers that can be used for basic,                                 |
+| <optimize>`                   |  NumPy-compatible quantum nodes, such as :func:`GradientDescentOptimizer`.           |
++-------------------------------+--------------------------------------------------------------------------------------+
+| :ref:`pennylane.qnode <qnode>`|  Defines the :class:`QNode` class.                                                   |
++-------------------------------+--------------------------------------------------------------------------------------+
+| :ref:`pennylane.templates     |  Contains the :mod:`templates.layers` and :mod:`templates.embeddings` submodules,    |
+| <templates>`                  |  which contain subroutines such as :func:`RandomCircuit`.                            |
++-------------------------------+--------------------------------------------------------------------------------------+
+
+Other modules
+-------------
+
+The following modules are contain lower-level functionality, which is more relevant for developers:
```

Comment:
The following modules contain lower-level functionality, which is more relevant for developers:

Reply:
```suggestion
The following modules contain lower-level functionality, which is more relevant for developers:
```